src/index.js

```diff
@@ -20,6 +20,27 @@ enterBtn.addEventListener('click', (event) => {
   listCollection.read();
 });
 
+const clearBtn = document.querySelector('.clear-completed');
+clearBtn.addEventListener('click', () => {
+  const checks = document.querySelectorAll('input[type=checkbox]');
+  const updateItem = [];
+  checks.forEach((checkbox, i) => {
+    if (checkbox.checked) {
+      listCollection.data.forEach((item, index) => {
+        item.index = index;
+      });
+      updateItem.push(i);
+    }
+  });
+  const updateList = listCollection.data.filter((item, i) => !updateItem.includes(i));
+  updateList.forEach((item, index) => {
+    item.index = index;
```

Comment:
- [ ] [OPTIONAL] Kindly add `1` to the index so the tasks index starts with 1 when the clear completed function is invoked. You did a good job on the delete function. 🥸️